docs/dashboard/x-ray.md

```diff
@@ -0,0 +1,65 @@
+# XRay
+
+The Altis Dashboard provides a way to view all backend PHP requests for developer debugging tasks and investigations. This feature is called XRay, and is available via the XRay tab on each application.
+
+XRay traces provide performance profiling, database queries, remote requests and exposes all request data for each backend PHP request.
+
+
+## Searching XRay Requests
+
+![Example XRay Summary](../assets/xray-summary.png)
+
+When debugging requests in Altis Cloud, it's typical to find the specific HTTP request you want to debug. The XRay interface is limited to showing 500 HTTP requests at a time, so it's advices to narrow your search down by timeframe, URL, or IP address. The "Before" and "After" fields accept any human readable format, or absolute timestamps. URLs, IPs or other fields can be specified via the "Filter Express" search box. The Altis Dashboard provides contextual help via the "?" icon. Below are some common examples when using Filter Expressions:
+
+**Show all requests to a specific URL**
+```
+http.url = "https://example.com/my-url/"
+```
+
+**Show all requests to a URL containing a term**
+```
+http.url contains "wp-admin"
+```
+
+**Show all requests from a specific IP address**
+```
+http.clientip = 1.2.3.4
+```
+
+## Understanding an XRay trace
+
+Each HTTP request is called an "XRay trace" and provides many useful pieces of debugging information.
+
+### Flamegraphs
+
+![Example Flamegrapth](../assets/xray-flamegraph.png)
+
+XRay Flamegraphs provide you with a full performance view of the whole HTTP request. Flamegraphs make it very easy to understand why a specific request is slow, and what the call-stack is for all parts of the page load.
```

Comment:
I think with both full & whole there's some redundancy so maybe something like this:
```suggestion
XRay Flamegraphs provide you with a full performance profile of the HTTP request. Flamegraphs make it very easy to understand why a specific request is slow, and what the call-stack is for all parts of the page load.
```